pkg/asset/tls.go

```diff
@@ -115,7 +119,7 @@ func newAPIKeyAndCert(caCert *x509.Certificate, caPrivKey *rsa.PrivateKey, altNa
 
 	config := tlsutil.CertConfig{
 		CommonName:   "kube-apiserver",
-		Organization: []string{"kube-master"},
+		Organization: []string{orgSystemMasters},
```

Comment:
IMHO, we should use the `--kubelet-client-certificate` flag with a certificate that has `system:masters` instead. Should be the more secure approach.